packages/frontend/app/(dashboard)/course/[cid]/(settings)/settings/chatbot_questions/components/EditChatbotQuestionModal.tsx

```diff
@@ -158,8 +174,33 @@ const EditChatbotQuestionModal: React.FC<EditChatbotQuestionModalProps> = ({
 
     await API.chatbot.staffOnly
       .updateQuestion(cid, valuesWithId)
-      .then(() => {
+      .then(async () => {
         message.success('Question updated successfully')
+        const notify = (values as any).emailNotifyOnAnswerUpdate
+        if (notify) {
```

Comment:
```suggestion
        if (values.emailNotifyOnAnswerUpdate) {
```

Also, why not add `emailNotifyOnAnswerUpdate` to the FormValues type?